assets/main.js

```diff
@@ -472,3 +473,48 @@ anchors.forEach((a) => {
   a.setAttribute('target', '__blank');
   a.setAttribute('rel', 'noopener noreferrer');
 });
+
+// Insert SVGs and Classes for report covers on the about page
+function insertReportSVG () {
+    
+  // Find all elements tagged with the report-cover class
+  var reports = document.getElementsByClassName("report-cover");
+
+  // Loop through elements and attach an svg and class in a set of 4, and then reset the counter on the 5th|11th|etc set
+  if(reports.length >= 1) {
```

Comment:
@KevJ2021 is there a chance reports may ever be null? if so maybe a null check :p ?

Reply:
Thank you for the feedback, @richmungla! 

Yeah, Javascript is a bit weird at times, if you were to do this check `if(reports.length >= 0)` and reports was null, it would still return true even though null is not numeric: https://blog.campvanilla.com/javascript-the-curious-case-of-null-0-7b131644e274 , but because we're checking if reports is greater than or equal to 1, it wouldn't execute the body statement.

Aside from that though, because I'm using document.getElementsByClassName, an HTMLCollection is constructed, so .length would be accessible and return a numeric value (0 if empty) and would not be null or undefined. https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByClassName